olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -82,6 +82,10 @@ spec:
         displayName: Knative Kafka
         kind: KnativeKafka
         name: knativekafkas.operator.serverless.openshift.io
+        statusDescriptors:
+          - description: The version of Knative Kafka installed
+            displayName: Version
+            path: version
```

Comment:
This has since been removed and should go.

Reply:
sigh, rebase fun, my bad